python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -364,3 +364,6 @@ def save(
             required_fields_only=required_fields_only,
             pad_sequence=pad_sequence,
         )
+
+    def calibration(self, **kwargs):
```

Comment:
[Minor] @marjanhs Typically, it is best practice to name functions with verbs. So maybe `calibrate()`?

Reply:
changed